.github/workflows/pr-build.yml

```diff
@@ -40,4 +40,24 @@ jobs:
       - name: Build with Gradle
         uses: hypertrace/github-actions/gradle@main
         with: 
-          args: dockerBuildImages
\ No newline at end of file
+          args: dockerBuildImages
+
+      - name: Determine docker tag
+        id: tag
+        run: echo ::set-output name=tag::$(./gradlew -q printDockerImageDefaultTag | head -1)
+
+      - name: Scan docker image
+        uses: azure/container-scan@v0.1
+        with:
+          image-name: hypertrace/java:11
+        env:
+          DOCKLE_HOST: "unix:///var/run/docker.sock"
+        continue-on-error: true
```

Comment:
We should require this to pass in this repo IMO. This is not an application container, it's the base container, so there's never a time it's acceptable to release it with a known issue.

Reply:
okay ill edit that

Reply:
So the key part here is my last line - I think continue on error is fine as long as we make the scan result a required check (that way, the run completes error or not and we can view all the results).

Reply:
@aaron-steinfeld can you make it a required check? 

@senanjude0 we may have to ignore some of the vulnerabilities for which fix is not available yet.